default/stringtables/ru.txt

```diff
@@ -1890,6 +1890,15 @@ OPTIONS_DB_SECTION_MISC
 OPTIONS_DB_SECTION_NETWORK
 Сервер и сеть
 
+<<<<<<< HEAD
```

Comment:
You just committed unmerged changes.

You should always resolve conflicts before add and commit. 

Do you need instruction on how to handle this?